### PR TITLE
UI Automation in Windows Console: fix ConsoleUIATextInfo.collapse(end=True)

### LINKED ISSUE #9875: UIA in Windows consoles: Say all behaves incorrectly
### Steps to reproduce:
1. Turn on UIA in console
1. Open console
1. Press NVDA+up arrow twice fast
1. Press NVDA+down arrow

### Actual behavior:
NVDA walks backwards through the text, speaking one extra letter at a time.
```
>
f>
ef>
eef>
Reef>
\Reef>
s\Reef>
rs\Reef>
ers\Reef>
>
f>
ef>
eef>
Reef>
\Reef>
s\Reef>
rs\Reef>
ers\Reef>
C:\Users\Reef>
```

### Expected behavior:
Standard "say all" behaviour.

#### NVDA version:
Running from source on `threshold_py3_staging` 
`beta-17939,fd65a653` seems to freeze.

#### Windows version:
Windows 10 - 1903
Build: 18362.175

#### Other information:

Quite possibly related to the textUtils offsets work.

### PR DESCRIPTION
**Pull request by codeofdusk**
### Link to issue number:
Builds on #9614. Fixes #9875.

### Summary of the issue:
When collapsing a `ConsoleUIATextInfo`, the `end` keyword argument was not properly handled.

### Description of how this pull request fixes the issue:
This PR now passes the `end` argument through the `super` call in `collapse`.

### Testing performed:
Tested say all in review (numpad plus) and verified that it is functioning correctly.

### Known issues with pull request:
None.

### Change log entry:
None.

### CODE REVIEW

**Comment by codeofdusk** [comment]
Cc @feerrenrut @leonardder

**Review by feerrenrut — APPROVED** [review]
Thanks for looking at this @codeofdusk, I'll might hold off until tomorrow before merging this.

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**
